### PR TITLE
R4R: Fix the Features section of docs

### PR DESCRIPTION
**Pull request by shilei-wang**
ref：https://github.com/irisnet/irishub/issues/1376

### CODE REVIEW

File: docs/features/basic-concepts/genesis-file.md [hunk 1]
```diff
@@ -54,5 +54,5 @@ Each genesis state starts with a list of account balances. Social consensus on t
 ## Gentxs
 
 Gentxs contains the transaction set of creating validators in genesis block. 
-The IRIS SDK provides robust tools for bootstrapping the identities that will start chain via the gen-tx process. gen-tx or a Genesis Transaction are cryptographically signed transactions that are executed during chain initialization that generate a starting set of validators.
+The Cosmos SDK provides robust tools for bootstrapping the identities that will start chain via the `gen-tx` process. `gen-tx` or a Genesis Transaction are cryptographically signed transactions that are executed during chain initialization that generate a starting set of validators.
```
Comment (chengwenxi) [inline docs/features/basic-concepts/genesis-file.md]: Cosmos SDK -> IRISnet or iris

File: docs/features/basic-concepts/fee.md [hunk 2]
```diff
@@ -5,30 +5,34 @@ Specify the maximum gas limit by `--gas`.
...
+The total gas needed for executing the transaction is the sum of gas needed for every operations performed during executing the transaction. User should specify the maximum gas by `--gas` parameter. If maximum gas is not enough for executing the transaction, the transaction won't be executed successfully and all the fee will be returned to user's account. After the transaction being executed successfully, fee will deduct according to gas used, deducted fee will be  maximum fee * gas used / maximum gas and left fee will be returned to user. Gas price is equal to maximum fee / maximum gas and stands for fee that user want to pay for each unit of gas. To keep the fee is set in a reasonable range, we set a minimum limit for gas price, 6*10^(-6) iris/gas, transaction won't be executed if the gas price is less than this value.
```
Comment (shilei-wang) [inline docs/features/basic-concepts/fee.md]: 2^(-8) iris/gas  --->6*10^(-6) iris/gas

File: docs/features/stake.md [hunk 3]
```diff
@@ -8,19 +8,24 @@ This specification briefly introduces the functionality of stake module and what
...
+	The quantity of validators can't increase indefinitely. Too many validators may result in low efficient consensus which slows down the blockchain TPS. So Byzantine-fault-tolerant POS blockchain network will have a limiation to the validator quantity. Usually, the value is 100. If more than 100 full nodes apply to join validator set. Then only these nodes with top 100 most bonded tokens will be real validators. Others will be validator candidates and will be descending sorted according to their bonded token amount. Once the one or more validators are kicked out from validator set, then the top candidates will be added into validator set automatically.
```
Comment (shilei-wang) [inline docs/features/stake.md]: The quantity of validators can't increase without limit. -->The quantity of validators can't increase indefinitely.

Comment (HaoyangLiu) [reply docs/features/stake.md]: The quantity of validators can't increase infinitely.

File: docs/features/bank.md [hunk 4]
```diff
@@ -7,30 +7,36 @@ This module is mainly used to transfer coins between accounts、query account ba
...
     ```bash
-    iriscli bank send --to [address of wallet B] --amount=10iris --fee=0.3iris --from=[key name of wallet A] --chain-id=[chain-id]
+    iriscli bank send --to=<address_of_wallet_B> --amount=10iris --fee=0.3iris --from=<key_name_of_wallet_A> --chain-id=<chain-id>
     ```
     IRISnet supports multiple tokens in circulation, and in the future IRISnet will be able to include multiple tokens in one transaction -- tokens can be any coin_type registered in IRISnet. 
 
 4. Sign transactions generated offline
 
-    To improve account security, IRISnet supports offline signing of transactions to protect the account's private key. In any transaction, you can build an unsigned transaction using the flag --generate-only=true. Use transfer transactions as an example:
+    To improve account security, IRISnet supports offline signing of transactions to protect the account's private key. In any transaction, you can build an unsigned transaction using the flag --generate-only. Use transfer transaction as an example:
```
Comment (HaoyangLiu) [inline docs/features/bank.md]: `Use transfer transaction as an example:` -> `Take transfer transaction for example:`

File: docs/features/basic-concepts/fee.md [hunk 5]
```diff
@@ -5,30 +5,34 @@ Specify the maximum gas limit by `--gas`.
...
+The resource needed for every transactions are varied for different type of transactions. For example, only a few computations, queries & modifies are needed for transfer transaction. But more computations, queries & modifies are needed for creating a validator.  Gas is the unit used to measure how much resource needed to execute the transaction. We list the gas needed for some typical operations in the below:
 
-- gas needed  for writing the tx to the database is: 10 * the size of the transaction data (in bytes)
+- gas needed for writing the tx to the database is: 10 * the size of the transaction data (in bytes)
 - gas needed for reading some data from database: 10 + data length(in bytes)
 - gas needed for writing some data to database: 10 + 10 * data length(in bytes)
-- sign or verify a signature
+- sign or verify a signature : 100
```
Comment (HaoyangLiu) [inline docs/features/basic-concepts/fee.md]: gas needed for signature verification: 100

File: docs/features/basic-concepts/fee.md [hunk 6]
```diff
@@ -5,30 +5,34 @@ Specify the maximum gas limit by `--gas`.
...
+- ....
```
Comment (HaoyangLiu) [inline docs/features/basic-concepts/fee.md]: Maybe a table for these gas consumption is better

File: docs/features/basic-concepts/genesis-file.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ Each genesis state starts with a list of account balances. Social consensus on t
 
 * **block_size** 
   * `max_bytes` The max size of a block
-  * `max_gas`  The maximum Gas quantity of a block. Its default value is -1 which means no gas limit. If the accumulation of comsumed gas exceeds the block gas limit, the transaction and all subsequent transactions in the same block will fail to deliver. 
+  * `max_gas`  The maximum Gas quantity of a block. Its default value is -1 which means no gas limit. If the accumulation of consumed gas exceeds the block gas limit, the transaction and all subsequent transactions in the same block will fail to deliver.
```
Comment (HaoyangLiu) [inline docs/features/basic-concepts/genesis-file.md]: `consumed gas` -> `gas consumption`

File: docs/features/basic-concepts/gov-params.md [hunk 8]
```diff
@@ -1,15 +1,14 @@
...
-parameters, it is available to put up a `parameter-change` proposal in governance module.
+All the IRIS holders are able to modify. If the community is not satisfied with certain modifiable parameters, it is available to submit a `parameter-change` proposal in governance module.
 
 ## Parameters in Auth
 
 | key |Description | Range|
 |----| ---|---|
 |`auth/gasPriceThreshold`  |minimum of gas price |(0, 10^18iris-atto]
-|`auth/txSizeLimit`  |the limit of the normal txsize |[500, 1500]
+|`auth/txSizeLimit`  |the limit of the normal tx size |[500, 1500]
```
Comment (HaoyangLiu) [inline docs/features/basic-concepts/gov-params.md]: limit -> limitation

File: docs/zh/features/distribution.md [hunk 9]
```diff
@@ -5,64 +5,167 @@
 该模块负责将收集的交易费和通胀的代币分发给所有验证人和委托人。为了减少计算压力，引入了一种被动分配策略。
 `被动`意味着不会直接向贡献者自动支付利益。委托人或者验证人必须手动发送交易以取回其收益，否则，他们的收益将保留在全局收益池中。
 
-每个出块人会获得该区块收到的手续费和通胀Token的一部分作为出块奖励。计算方法：
+## 收益
+
+### 收益的来源
+
+1.交易所产生的交易费 `fee` (由交易的第一签名者支付`fee`)
```
Comment (chengwenxi) [inline docs/zh/features/distribution.md]: 第一个签名者

File: docs/zh/features/distribution.md [hunk 10]
```diff
@@ -5,64 +5,167 @@
...
+验证人自己委托部分的委托收益
+验证人节点所有委托所产生的佣金
+
+2.委托人
+
+委托人的委托收益
+
+例子：
+```
+假设： 当前委托收益为 4iris（委托10000iris一天产生的总收益，与实际较为接近）， 验证人节点设置佣金比例 10% ，
```
Comment (chengwenxi) [inline docs/zh/features/distribution.md]: 万份日收益为 4iris

File: docs/features/guardian.md [hunk 11]
```diff
@@ -5,32 +5,36 @@ IRISnet introduce two types of privileged system user controlled by foundations,
...
+    2. Invocate a service by `profiling` mode, under which service fees can be exempted.
     
 * Trustee privileges
     1. To be the destination address if the usage type of a `TxTaxUsage` proposal is `Distribute` or `grant`.
-    2. Send transaction to withdraw coins to an account from system service fee tax pool.
+    2. Send `withdraw-tax` transaction to withdraw coins to an account from system service fee tax pool.
+    
+* Genesis Profiler (Defined in genesis.json)
+    1. Only Genesis Profiler can add/delete Ordinary Profiler account
+    2. Only Genesis Profiler can add/delete Trustee account
```
Comment (chengwenxi) [inline docs/features/guardian.md]: Only Genesis Trustees can add/delete Ordinary Trustee account

File: docs/zh/features/guardian.md [hunk 12]
```diff
@@ -1,27 +1,31 @@
...
-* Trustee的特权
-    1. 如果`TxTaxUsage`提议的使用类型是`Distribution`或`Grant`，作为目的地地址。
-    2. 发送交易从系统服务费税池中提取代币到账户。
+* Trustee的权益
+    1. 通过`TxTaxUsage`治理取回交易税费时，只能使用Trustee address作为取回地址。
+    2. 发起`withdraw-tax`交易可以从`iService`服务费税池中提取代币到指定账户。
+    
+* Genesis Profiler的权益（在创世的genesis.json中定义）
+    1. 只有Genesis Profiler可以 添加/删除 普通Profiler账户
+    2. 只有Genesis Profiler可以 添加/删除 Trustee账户
```
Comment (chengwenxi) [inline docs/zh/features/guardian.md]: 只有Genesis Trustees可以 添加/删除 普通Trustee账户

File: docs/features/basic-concepts/genesis-file.md (hunk 1 above)
Comment (shilei-wang) [reply docs/features/basic-concepts/genesis-file.md]: fixed

File: docs/features/stake.md (hunk 3 above)
Comment (shilei-wang) [reply docs/features/stake.md]: fixed

File: docs/features/basic-concepts/fee.md (hunk 5 above)
Comment (shilei-wang) [reply docs/features/basic-concepts/fee.md]: fixed

File: docs/features/basic-concepts/fee.md (hunk 6 above)
Comment (shilei-wang) [reply docs/features/basic-concepts/fee.md]: Good suggestion

Comment (shilei-wang) [reply docs/features/basic-concepts/fee.md]: fixed

File: docs/features/bank.md (hunk 4 above)
Comment (shilei-wang) [reply docs/features/bank.md]: fixed

File: docs/features/basic-concepts/genesis-file.md (hunk 7 above)
Comment (shilei-wang) [reply docs/features/basic-concepts/genesis-file.md]: fixed

File: docs/features/basic-concepts/gov-params.md (hunk 8 above)
Comment (shilei-wang) [reply docs/features/basic-concepts/gov-params.md]: fixed

File: docs/zh/features/distribution.md (hunk 10 above)
Comment (shilei-wang) [reply docs/zh/features/distribution.md]: fixed

File: docs/zh/features/distribution.md (hunk 9 above)
Comment (shilei-wang) [reply docs/zh/features/distribution.md]: fixed

File: docs/features/guardian.md (hunk 11 above)
Comment (shilei-wang) [reply docs/features/guardian.md]: fixed

File: docs/zh/features/guardian.md (hunk 12 above)
Comment (shilei-wang) [reply docs/zh/features/guardian.md]: fixed

File: docs/features/service.md [hunk 13]
```diff
@@ -24,68 +24,68 @@ The following parameters can be modified by governance(governance.md)
...
+If the service consumer needs to initiate a service invocation request, the service fee specified by the service provider needs to be paid. The service provider needs to respond to the service request within the block height defined by `MaxRequestTimeout`. If the service provider does not respond in time, the deposit of the 'SlashFraction' ratio will be deducted from the service provider's service binding deposit. If the service call is responded normally, the system will deduct the `ServiceFeeTax` ratio from the service fee, and add the remaining service fee to the service provider's incoming pool. If the service call does not respond in time, the service fee will be refunded to the service consumer's return pool. The service provider/consumer can initiate the `withdraw-fees`/`refund-fees` transaction to retrieve all of the tokens in the incoming/return pool.
```
Comment (chengwenxi) [inline docs/features/service.md]: If the service provider does not respond in time, the deposit of the 'SlashFraction' ratio will be deducted from the service provider's service binding deposit, and the service fee will be refunded to the service consumer's return pool.

Comment (shilei-wang) [reply docs/features/service.md]: fixed

**Comment by haifengxi** [comment]
Thanks @HaoyangLiu for the detailed review!

**Pull request merged**